style.css

```
@@ -1 +1,56 @@
 /*  Add your styling here */
+body  {
+    display: grid;
+    grid-template-areas: 1fr 2fr 4fr 3r;
+    background-color: #ffe6ff;
+}
+
+.nav-bar > ul {
+   display: inline;
+   padding: 100px 6em 6em 7em;
+   font-size: 1.25em;
+}
+
+.title, .tagline {
+    text-align: center;
+}
+
+.title {
+    font-size: 3em;
+    padding: 1em 1em;
+}
+.tagline {
+    font-size: 2em;
+    padding: .75em;
+}
+
+.order-now {
+    border-radius: 55%;
+    padding-bottom: 2em;
+    font-size: 1em;
+}
+
+.info {
+    text-align: center;
+    background-color: #ff80ff;
+}
+
+.cake-images {
```

```suggestion
.cake {
```

I don't think this rule can be applying as there's no cake-images class in the HTML that I can see. Devtools should help you find and fix these errors.

https://developer.chrome.com/docs/devtools/